src/library_webgl2.js

```diff
@@ -723,21 +723,41 @@ var LibraryWebGL2 = {
     GL.syncs[id] = null;
   },
 
+#if WASM_BIGINT
+  glClientWaitSync__sig: 'iiii',
```

Comment:
I think you can just use `iiij` and avoid the different signatures.

Reply:
Ah right, forgot about that one. That works well.